tests/console/ansible.pm

```diff
@@ -100,12 +114,20 @@ sub run {
     die 'network-engine should be installed!' unless $galaxy_installed =~ m/ansible-network\.network-engine/;
 
     # 4. Ansible playbook testing
+    my $skip_tags;
+    $skip_tags = '--skip-tags zypper' if is_transactional;
 
     # Check that community.general.zypper module is available
     assert_script_run 'ansible-doc -l community.general | grep zypper';
 
+    # Check the version of ansible-community from where we use the zypper module
+    script_run 'ansible-community --version';
```

Comment:
This command seems to fail

* https://pdostal-server.suse.cz/tests/2811#step/ansible/142
* https://pdostal-server.suse.cz/tests/2812#step/ansible/97

Reply:
It's expected for older Ansible versions. I added comment.